app/models/cart.rb

```diff
@@ -4,13 +4,19 @@ class Cart
   def initialize(initial_contents)
     @contents = initial_contents || {}
   end
-
+  
   def add_beer(beer, quantity)
-    if contents.has_key?(beer.id)
-      contents[beer.id][:quantity] += quantity
-      contents[beer.id][:beer_price] = beer.price * contents[beer.id][:quantity]
+    if contents.has_key?(beer.id.to_s)
+      contents[beer.id.to_s]["quantity"] = contents[beer.id.to_s]["quantity"].to_i + quantity.to_i
+      contents[beer.id.to_s]["beer_price"] = beer.price * contents[beer.id.to_s]["quantity"].to_i
     else
-      contents[beer.id] = {beer_name: beer.name, beer_price: beer.price, quantity: quantity} 
+      contents[beer.id.to_s] = {"beer_name" => beer.name, "beer_price" => beer.price, "quantity" => quantity}
```

Comment:
Line is too long. [109/80]<br>Space inside { missing.<br>Space inside } missing.